docs/reference/commandline/secret_ls.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "secret ls"
+description: "The secret ls command description and usage"
+keywords: ["secret, ls"]
+---
+
+<!-- This file is maintained within the docker/docker Github
+     repository at https://github.com/docker/docker/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# secret ls
+
+```Markdown
+Usage:	docker secret ls [OPTIONS]
+
+List secrets
+
+Aliases:
+  ls, list
+
+Options:
+  -q, --quiet          Only display IDs
+```
+
+This command when run targeting a manager, lists secrets in the
```

Comment:
Maybe, just say "Run this command from a manager to list the secrets in the swarm."... or something like that.